spec/features/invoices/show_spec.rb

```diff
@@ -43,4 +43,11 @@
       expect(page).to have_content("#{invoice_item.status}")
     end
   end
+
+  it 'I see the total revenue that will be generated from all of my items on the invoice' do
+    visit merchant_invoice_path(@merchant_1, @invoice_4)
+    expect(page).to have_content("Total Revenue")
+    # expect(page).to have_content(@invoice_4.total_revenue)
```

Comment:
This test looks like it should be updated to check for the total revenue. Would be good to uncomment line 50/update the test a little.